Example/ConsentViewController_ExampleTests/MessageWebViewControllerSpec.swift

```diff
@@ -133,7 +133,7 @@ class MessageWebViewControllerSpec: QuickSpec, GDPRConsentDelegate, WKNavigation
                             "body": ["type": 12, "id": "id", "payload": ["pm_url": "pm_url"]]
                         ])
                         messageWebViewController.userContentController(userContentController, didReceive: message)
-                        let expectedPMURL = "https://notice.sp-prod.net/privacy-manager/index.html?site_id=1&consentUUID=uuid&message_id=1234"
```

Comment:
Same here. We should check only for the query param, not the entire URL. Furthermore, I think it'd be more descriptive if we changed:
```swift
it("calls the onAction on the consent delegate with ShowPrivacyManager with wrong PM URL")
```
to
```swift
context("and the pm_url attribute is invalid") {
  it("uses its own pmId as message_id" {
     // test here
  }
}
```
This way, the `context` gives the pre condition, and the `it` explains what should happen.

Reply:
@andresilveirah updated the spec.